libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html

```diff
@@ -1,32 +1,94 @@
-<div *ngIf="shownOrganization" class="mb-6 sm:mb-12">
-  <p class="text-gray-700 text-xs mb-3 uppercase" translate>
-    record.metadata.contact
-  </p>
-  <div class="flex mb-1">
+<div class="py-4 px-5 rounded bg-gray-100 text-black">
+  <div *ngIf="shownOrganization" class="mb-6 sm:mb-12">
+    <p class="text-sm mb-3 font-medium" translate>record.metadata.contact</p>
     <div
-      class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
-      (click)="onOrganizationClick()"
+      *ngIf="contacts[0].organization?.logoUrl?.href"
+      class="mb-3 flex items-center justify-center border-solid border border-gray-300 rounded-md bg-white h-32"
```

Comment:
this needs an `overflow-hidden` class otherwise there is a "cut edges" effect on logos, see:
![image](https://github.com/geonetwork/geonetwork-ui/assets/10629150/04492e3f-c134-4a25-afc1-024decc7d99a)

Reply:
Good catch! Thanks :)